### DISCUSSION TITLE
Addressing button/menu via script?

### DISCUSSION THREAD

**Discussion by travisfranck** [joelekstrom/fastmate#74] [Q&A]
I'd like to find a way to use a keyboard shortcut to change the Alias that I'm sending from.  I thought I might be able to use javascript to set the button value of the Alias menu in the compose view, but the ID for this button seems to be dynamic.  

What is the best way to script portions of the Fastmail UI?  Specifically the Aliases menu in the Compose view.

**Comment by joelekstrom** [comment]
Hi! I think you're on to something there - regarding the id (in my case something like `v373`), they indeed _seem_ dynamic, but Fastmate also relies on them for some functionality. They seem somewhat stable. I have to update some times but it has not been often, so I think you can go that route. I can't think of a better way at the moment but I'll think about it 🤔 

As for the shortcut itself, not sure you can capture a keypress in JS, but it should work. I've been considering adding a Fastmate JS API where you can register handlers in a userscript which will show up as actions in the Fastmate menu bar, with custom shortcut support.
